apps/financial-aid/web-osk/src/components/StatusLayout/StatusLayout.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode, useContext, useEffect, useMemo } from 'react'
+import { Box, GridContainer, Text } from '@island.is/island-ui/core'
+
+import * as styles from './StatusLayout.treat'
+
+import {
+  LogoHfj,
+  Login,
+  HasApplied,
+} from '@island.is/financial-aid-web/osk/src/components'
+
+import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
+import { ApplicationState, getState } from '@island.is/financial-aid/shared'
+
+interface Props {
+  children: ReactNode
+}
+
+const StatusLayout = ({ children }: Props) => {
+  const { isAuthenticated, user } = useContext(UserContext)
+
+  const currentState = useMemo(() => {
+    if (user?.activeApplication) {
+      return getState[user.activeApplication[0].state]
+    }
+  }, [user])
+
+  useEffect(() => {
+    document.title = 'Fjárhagsaðstoð – Staða'
+  }, [])
+
+  if (isAuthenticated === false) {
+    return <Login />
+  }
+  if (!user) {
+    return null
+  }
+
+  return (
+    <Box
+      paddingY={[3, 3, 3, 6]}
+      background="purple100"
+      className={styles.processContainer}
+    >
+      {user.activeApplication?.length === 0 ? (
```

Comment:
We need to change some logic here in the near future, when we get either the id from the url or current application from context. Just an fyi. 

We can null check this when we change the contract.